recipes/tables-io/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "tables-io" %}
+{% set version = "0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace("-", "_") }}-{{ version }}.tar.gz
+  sha256: 2b98e4edc3c2fb4ca2452c7e8e3267bdb96b0a690d9fcf79d5ece3b96f7c99ad
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  noarch: python
+
+requirements:
+  host:
+   - python
+   - pip
+   - setuptools_scm
+
+  run:
+    - python >=3.6
+    - numpy >1.21
```

Comment:
This requirement is really harsh. Does this package really need a numpy that is nearly bleeding edge?

Reply:
I read through the source code and all that is being called are things like `np.shape`, `np.ndim` etc. This pin is not needed and it is disruptive.